CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Fixed
 * Wrong generic constraint placement in ML-style generic definitions with multiple type parameters. [#1868](https://github.com/fsprojects/fantomas/issues/1868)
 
+### Documented
```

Comment:
`Documented` doesn't seem to be listed on https://keepachangelog.com/en/1.0.0/
https://github.com/ionide/KeepAChangelog#changelogdata

I might remove this later.